drivers/hmis/app/graphql/mutations/create_project_config.rb

```diff
@@ -18,16 +20,7 @@ def resolve(input:)
       errors.add :config_type, :required if input.config_type.blank?
       return { errors: errors } if errors.any?
 
-      class_name = case input.config_type
-      when 'AUTO_ENTER'
-        Hmis::ProjectAutoEnterConfig
-      when 'AUTO_EXIT'
-        Hmis::ProjectAutoExitConfig
-      when 'STAFF_ASSIGNMENT'
-        Hmis::ProjectStaffAssignmentConfig
-      else raise "Unsupported type: #{input.config_type}"
-      end
-
+      class_name = input.config_type.constantize
```

Comment:
This is a breaking change but I think it's OK since the project settings page is only used by a couple people and sparingly